app/api/services/suppliers.py

```diff
@@ -276,7 +332,8 @@ def get_approved_suppliers(self):
                 Supplier.data['recruiter'].astext.label('recruiter_status'),
                 Supplier.data['seller_type'].label('seller_type'),
                 Supplier.data['number_of_employees'].label('number_of_employees'),
-                func.string_agg(Domain.name, ',').label('domains')
+                func.string_agg(Domain.name, ',').label('domains'),
+                Supplier.data['labourHire'].label('labour_hire')
```

Comment:
I've just opened #747 which fixes a bug when this query returns null for `seller_type`. It looks like the csv code will be fine but worth a double check.

Reply:
Yep - I can't see `seller_type` being used in the CSV code so it should be okay.